templates/sql.tftpl

```diff
@@ -0,0 +1,131 @@
+locals {
+  sql_tags = {
```

Comment:
Should SQL use entirely different tags than our default? If SQL requires additional tags it should be merged with the default tags